From 6b827b2811eaaf8f269f99c7d8cf0f7d2dc83fb9 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:06:22 -0400
Subject: [tools] perf: Use LDFLAGS instead of ALL_LDFLAGS

Message-id: <1410181752-25631-138-git-send-email-jolsa@redhat.com>
Patchwork-id: 91786
O-Subject: [PATCH RHEL7.1 BZ1134356 137/307] perf tools: Use LDFLAGS instead of ALL_LDFLAGS
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit e27a08f53d081b33c988db954008d879b2ab8b39
Author: Jiri Olsa <jolsa@kernel.org>
Date: Sun Apr 27 11:12:21 2014 +0200

description
===========
We no longer use ALL_LDFLAGS, Replacing with LDFLAGS.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Makefile.perf b/tools/perf/Makefile.perf
index e969233..895edd3 100644
--- a/tools/perf/Makefile.perf
+++ b/tools/perf/Makefile.perf
@@ -589,7 +589,7 @@ $(GTK_OBJS): $(OUTPUT)%.o: %.c $(LIB_H)
  $(QUIET_CC)$(CC) -o $@ -c -fPIC $(CFLAGS) $(GTK_CFLAGS) $<
 
 $(OUTPUT)libperf-gtk.so: $(GTK_OBJS) $(PERFLIBS)
- $(QUIET_LINK)$(CC) -o $@ -shared $(ALL_LDFLAGS) $(filter %.o,$^) $(GTK_LIBS)
+ $(QUIET_LINK)$(CC) -o $@ -shared $(LDFLAGS) $(filter %.o,$^) $(GTK_LIBS)
 
 $(OUTPUT)builtin-help.o: builtin-help.c $(OUTPUT)common-cmds.h $(OUTPUT)PERF-CFLAGS
  $(QUIET_CC)$(CC) -o $@ -c $(CFLAGS) \
-- 
1.7.1